server/config.json

```diff
@@ -1,11 +1,11 @@
 {
   "dashboard": {},
-  "pg_url": "postgres:///shovel",
+  "pg_url": "$DATABASE_URL",
   "eth_sources": [
     {
-      "name": "sepolia",
-      "chain_id": 11155111,
-      "url": "https://rpc2.sepolia.org",
+      "name": "base_sepolia",
+      "chain_id": 84532,
+      "url": "https://base-sepolia.infura.io/v3/049a0c3141de47f1afcbc640911be2a7",
```

Comment:
got a new one, `https://base-sepolia.g.alchemy.com/v2/${API_KEY}`. 
Where is `$DATABASE_URL` stored, is it env? could put the `API_KEY` too there

Reply:
whoops didn't realize that was a key but makes sense